src/redjubjub.rs

```diff
@@ -80,6 +80,8 @@ impl<E: JubjubEngine> PublicKey<E> {
         let c = h_star::<E>(&rbar[..], msg);
 
         // S . G = R + c . vk
+        // The other signature validation checks are performed
```

Comment:
Please specify what you mean. 'Specifically, the field type enforces the range of the scalars' or whatever is actually going to be true. If I'm an auditor diffing this against the RFC, I'll be looking for specific mention of the preconditions. If I don't see it, I might waste engagement time chasing a malleability red herring.